### PR TITLE
Support for GTFS transfers' accessibility data

### PR DESCRIPTION
**Pull request by lboyarsky**
Ticket: [ensure that Sullivan to Andrew step-free trip goes through Park St elevator](https://app.asana.com/0/810933294009540/1114088692253436)

**Before**:
image

**After**:
image

### CODE REVIEW

File: src/main/java/org/opentripplanner/model/Transfer.java [hunk 1]
```diff
@@ -25,6 +25,10 @@
 
     private int minTransferTime = MISSING_VALUE;
 
+    private int minWheelchairTime = MISSING_VALUE;
+
+    private int wheelchairTransfer;
```
Comment (paulswartz) [inline src/main/java/org/opentripplanner/model/Transfer.java]: This looks duplicated from the other PR?

Comment (lboyarsky) [reply src/main/java/org/opentripplanner/model/Transfer.java]: Yes, but we need to extend both models -- OneBusAway model, which is used for GTFS parsing, and OpenTripPlanner's model, which is used for trip planning.

Comment (paulswartz) [reply src/main/java/org/opentripplanner/model/Transfer.java]: In that case, should this also get a default of 0?

Comment (lboyarsky) [reply src/main/java/org/opentripplanner/model/Transfer.java]: We only create instances of this class when mapping from OneBusAway model, so I'd say it shouldn't have the default (just like `transferType` field few lines above doesn't have it), because let's say we have different default values here and in OneBusAway library, it would be pretty confusing for the reader and anyway this second default value, which we're going to have here, is going to be ignored. 

To put it differently, in my point of view, having default makes sense when reading from csv (that is, default value is used when there was no value specified in csv), but not when doing model conversion (because defaults already have been applied to the model being converted).
